setup.py

```diff
@@ -53,6 +53,7 @@
             "SurfaceWithSeismicCrossSection = webviz_subsurface.plugins:SurfaceWithSeismicCrossSection",
             "WellCrossSection = webviz_subsurface.plugins:WellCrossSection",
             "WellCrossSectionFMU = webviz_subsurface.plugins:WellCrossSectionFMU",
+            "AssistedHistoryMatchingAnalysis = data_analysis_test1.plugins:AssistedHistoryMatchingAnalysis",
```

Comment:
```suggestion
            "AssistedHistoryMatchingAnalysis = webviz_subsurface.plugins:AssistedHistoryMatchingAnalysis",
```
...and you should then also add it to `webviz_subsurface/plugins/__init__.py` 🙂